### PR TITLE
feat: carbon angular index test

### LINKED ISSUE #514: Index Carbon Angular

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #514 

Indexing PR: 
https://github.com/IBM/carbon-components-angular/pull/2081

## Testing / reviewing
Run the app locally you should see `carbon components angular` 41 components with the following filters:
Screen Shot 2022-05-02 at 1 11 40 PM

### CODE REVIEW

File: services/web-app/data/libraries.js [hunk 1]
```diff
@@ -97,6 +97,14 @@ const prodLibraries = {
...
     group: carbonComponentsGroup
   },
+  // 'carbon-components-angular': {
+  //   host: 'github.com',
+  //   org: 'andreancardona',
+  //   repo: 'carbon-components-angular',
+  //   path: '/src',
+  //   group: carbonComponentsGroup,
+  //   ref: '514-index-carbon-angular'
+  // },
```
Comment (francinelucca) [inline services/web-app/data/libraries.js:107]:
```suggestion
  // 'carbon-components-angular': {
  //   host: 'github.com',
  //   org: 'carbon-design-system',
  //   repo: 'carbon-components-angular',
  //   path: '/src',
  //   group: carbonComponentsGroup
  // },
```

**Review by francinelucca — CHANGES_REQUESTED** [review]
comments on https://github.com/IBM/carbon-components-angular/pull/2081

**Review by francinelucca — COMMENTED** [review]
LGTM!

**Review by alisonjoseph — DISMISSED** [review]

**Review by alisonjoseph — DISMISSED** [review]

**Review by alisonjoseph — APPROVED** [review]

**Review by francinelucca — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**